oidnkt/src/main/kotlin/net/time4tea/oidn/Oidn.kt

```diff
@@ -144,15 +144,26 @@ class OidnFilter(private val ptr: Long) : AutoCloseable {
         jniCommit(ptr)
     }
 
+    fun executeTimed() {
+        timed("${javaClass.name}:execute") { execute() }
+    }
+
     fun execute() {
-        timed("${javaClass.name}:execute") { jniExecute(ptr) }
+        jniExecute(ptr)
     }
 
     fun setFilterImage(colour: FloatBuffer, output: FloatBuffer, width: Int, height: Int) {
         jniSetSharedFilterImage(ptr, "color", ensureDirect(colour), width, height)
         jniSetSharedFilterImage(ptr, "output", ensureDirect(output), width, height)
     }
 
+    fun setAdditionalImages(albedo: FloatBuffer, normal: FloatBuffer?, width: Int, height: Int) {
```

Comment:
The normal map is optional but when a normal map is used, the albedo map is required. That's why `normal` is optional but `albedo` isn't.